dev_guide/routes.adoc

```diff
@@ -133,6 +133,18 @@ TLS termination and a default certificate, which may not match the requested
 host name, resulting in validation errors.
 ====
 
+[[creating-routes-shards]]
+
+== Creating Routes with router shards
+
+In case us use the router sharding possibility 
+link: ../install_config/install/deploy_router.adoc#creating-router-shards[Creating Router Shards]
```

Comment:
- The `link:` syntax does not permit a space between `link:` and the URL.
- I suggest rewriting the paragraph (as explained above), and adding another link.